jobrunner/docker.py

```diff
@@ -25,6 +25,13 @@
 LABEL = "job-runner"
 
 
+# Shelling out to the Docker client on Windows sometimes hangs indefinitely
+# (for no obvious reason that we can determine). To prevent this locking up the
+# entire process we use a large-ish default timeout on almost all calls to
+# Docker.
+DEFAULT_TIMEOUT = 5 * 60
```

Comment:
I'm guessing this is seconds?  The subprocess.run docs aren't very clear here, could you add a comment to be explicit about it?

Reply:
Good call, thanks.